src/components/login/signup/SignupPolicyScreen.tsx

```
@@ -38,15 +39,15 @@ const SignupPolicyScreen = ({ toNextStage }: SignupPolicyScreenProps) => {
-              <span className="text-sm text-gray-400">이용 약관 동의 </span>
-              <span className="text-xs text-gray-400">(필수)</span>
+              <span className="typo-regular-2 text-gray-400">이용 약관 동의 </span>
+              <span className="typo-regular-4 text-gray-400">(필수)</span>
             </div>
           </div>
           <div>
             <a
               href="https://solid-connection.notion.site/1b5ac2cda9b28017a801d0cacf236ec7?pvs=4"
               target="_blank"
-              className="text-xs text-gray-400 underline"
+              className="typo-regular-4 text-gray-400 underline"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)